src/main/net/polydawn/mdm/commands/MdmStatusCommand.java

```diff
@@ -52,6 +57,17 @@ public MdmExitMessage call() throws IOException {
 			assertInRepo();
 		} catch (MdmExitMessage e) { return e; }
 
+		if (depName != null) {
```

Comment:
A large branch here makes me a little nervous.  I wonder if it's better to reuse the `MdmModuleSet` path to do the whole load, and then just filter out the relevant component of that?